pages/search.js

```diff
@@ -11,7 +11,7 @@ import Input from "@components/form/Input";
 import { getTags } from "./api/discover/tags";
 import { getProfiles } from "./api/profiles";
 import Pagination from "@components/Pagination";
-import { PROJECT_NAME } from "@constants/index";
+
```

Comment:
Facing bad update branch and fix conflicts by me

```suggestion
import { PROJECT_NAME } from "@constants/index";

```